### PR TITLE
Avoid case reports triggering the loading spinner

### LINKED ISSUE #2806: Spinner does not stop on download
Confirmed in Chrome and Safari. No document ready event after download completes supposedly. Lets check what is triggered then!

### PR DESCRIPTION
**Pull request by northwestwitch**
Fix #2806

**How to test**:
- Locally, on master branch case page, try to download the PDF delivery report or the PDF general report (RD case) or the Coverage & QC report (cancer case). After they are downloaded the loading spinner won't stop spinning on case page
- Switch to this branch and the loading spinner won't be triggered by the file download

**Expected outcome**:
The functionality should be working
Take a screenshot and attach or copy/paste the output.

**Review:**
- [ ] code approved by
- [ ] tests executed by

### CODE REVIEW

**Comment by northwestwitch** [comment]
Ready for review. This should fix the spinner when downloading of 3 PDF reports. Are there any more situations like this?

File: scout/server/blueprints/cases/templates/cases/collapsible_actionbar.html [hunk 1]
```diff
@@ -75,7 +75,7 @@
                 <i class="fa fa-link"></i></a>
         </div>
         <div>
-          <a href="{{ url_for('cases.pdf_case_report', institute_id=institute._id, case_name=case.display_name) }}">
+          <a href="{{ url_for('cases.pdf_case_report', institute_id=institute._id, case_name=case.display_name) }}" target="_blank">
```
Comment (dnil) [inline scout/server/blueprints/cases/templates/cases/collapsible_actionbar.html:78]: 👍

**Review by dnil — APPROVED** [review]
Super! Dont forget the vcf2cytosure one as well!

**Comment by northwestwitch** [comment]
We still have to fix the spinner when the user exports the variant list from variants filters. That one is a bit tricky because it's not a link but a form submit element.

**Comment by dnil** [comment]
Ah, right - good call!

**Comment by dnil** [comment]
How about just setting the form target to "_blank"? Does it break anything or open an annoying page?

**Comment by northwestwitch** [comment]
> How about just setting the form target to `"_blank"`? Does it break anything or open an annoying page?

It opens the annoying page!

**Pull request merged**